src/js/stores/transforms/AppFormFieldToModelTransforms.js

```diff
@@ -140,6 +140,9 @@ const AppFormFieldToModelTransforms = {
       .map(portDefinition => {
         var definition = Object.assign({}, portDefinition);
         definition.port = parseInt(definition.port, 10);
+        if (definition.name != null) {
```

Comment:
Oh! :)

This ternary makes an unnecessary assignment.

Just this block does the trick - even without the null-check:
```JavaScript
if (definition.name === "") {
  definition.name = null;
}
```